resourcematch-app/src/pages/index/index.page.tsx

```diff
@@ -30,15 +30,15 @@ function Page() {
     setResponses([])
     const urls: string[] = await callMatchApi(input)
     if(urls.length === 0){
-      setNoResults(true)
+      setError('No resources found')
     }else{
-      setNoResults(false)
+      setError('')
     }
     setResponses(urls)
   }
 
   const callMatchApi = async (input: string) => {
```

Comment:
Minor: It might be nice to specify the return type explicitly here:

```javascript
const callMatchApi = async (input: string): Promise<string[]> => {
```